Presentation.Web/Models/API/V2/Internal/Response/Notifications/NotificationResponseDTO.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using Presentation.Web.Models.API.V2.Types.Notifications;
+
+namespace Presentation.Web.Models.API.V2.Internal.Response.Notifications
+{
+    public class NotificationResponseDTO
+    {
+        /// <summary>
+        /// Uuid of the Notification
+        /// </summary>
+        public Guid Uuid { get; set; }
+        /// <summary>
+        /// Indicates whether notification is active
+        /// </summary>
+        public bool Active { get; set; }
+        /// <summary>
+        /// Name of the notification (different field than Subject)
+        /// </summary>
+        public string Name { get; set; }
+        /// <summary>
+        /// Date when the notification was sent last time
+        /// </summary>
+        public DateTime? LastSent { get; set; }
+        /// <summary>
+        /// Date when the notification becomes active
+        /// </summary>
+        public DateTime? FromDate { get; set; }
+        /// <summary>
+        /// Date when the notification expires
+        /// </summary>
+        public DateTime? ToDate { get; set; }
+        /// <summary>
+        /// Subject of the email
+        /// </summary>
+        public string Subject { get; set; }
+        /// <summary>
+        /// Body of the email
+        /// </summary>
+        public string Body { get; set; }
+        /// <summary>
+        /// Resource owning the notification (e.g. ItContract)
+        /// </summary>
+        public OwnerResourceType Type { get; set; }
+        /// <summary>
+        /// Type of the notification (Immediate/Scheduled)
+        /// </summary>
+        public NotificationSendType NotificationType { get; set; }
+        /// <summary>
+        /// Uuid of the resource owning the notification
+        /// </summary>
+        public Guid OwnerResourceUuid { get; set; }
```

Comment:
By convention we use identityname,,,, dtos for cross references in responses, to let's make it a "OwnerResource" property of that type